Build: Publish typings on travis publish (#1517)
styleguidist/react-styleguidist: docs/Cookbook.md

}
```

## How to re-use the types in Styleguidist?

> From version 10, styleguidist is written using typescript language.

Reviewer: ```suggestion
From version 10, Styleguidist is written using TypeScript language.
```

It allows the maintainers to catch type mismatch before execution and a better developer experience.

> It also allows users to wtite their customized styleguide components using typescript TSX and plugins to build it.

Reviewer 2: typo `write`

Author: good catch thank you

> It also allows users to write their customized styleguide components using typescript TSX and plugins to build it.

Reviewer: ```suggestion
It also allows users to write their customized style guide components using TypeScript and plugins to build it.
```

What plugins do you mean here?

Author: I meant to leverage the type checking powers of typescript and its red squigglies to help you code.
Plugins would be vscode type checking, or Sublime Text or Atom TS plugin.

I think I can omit it as it is self explanatory.

What do you think?

Reviewer: Yup, that's quite confusing for me ;-)

Author: I updated it.

> There is a trick though. Since all files in `src/client/rsg-components` are aliased to `rsg-components` using webpack, you will have to add this alias to your `tsconfig.json` file

Reviewer: ```suggestion
There is a trick though. Since all files in `src/client/rsg-components` are aliased to `rsg-components` using webpack, you will have to add this alias to your `tsconfig.json` file:
```

> Here is an example of what this file could contain:

Reviewer: ```suggestion
```
